### PR TITLE
Style links based on the type of link

### PR DESCRIPTION
**Pull request by Ray2126**
Designs based off https://www.figma.com/file/IgRlQWJTyGoZhlF4fVKD6B/Tailwind-Centrapay-Docs?node-id=2%3A2&t=PG8pI9FLaUsF6yv9-0

image

Test plan:
- Navigate to https://docs.centrapay.com/guides/farmlands-pos-integration/ and check that links look like the screenshot above

### CODE REVIEW

File: components/Prose.vue [hunk 1]
```diff
@@ -6,7 +6,6 @@
       prose-h2:scroll-mt-16 prose-h2:mt-8 prose-h2:pt-8 prose-h2:border-t
       prose-h3:scroll-mt-20
       prose-lead:text-slate-500
-      prose-a:no-underline
```
Comment (MeganSteenkamp) [inline components/Prose.vue:9]: There is a regression here with our markdown headings which are also links:
Screenshot 2023-01-18 at 9 21 24 AM

File: components/icons/LinkExternal.vue [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<template>
```
[the remark this answers is not in the document]
Comment (Ray2126) [reply components/icons/LinkExternal.vue]: Yep, removed the component I added and replaced it with the existing one.

File: components/content/ProseA.vue [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<template>
+  <NuxtLink
+    :to="props.href"
+    class="no-underline hover:no-underline inline-flex justify-center items-center"
```
Comment (MeganSteenkamp) [inline components/content/ProseA.vue:4]: We want an external link to open in a new tab so you'll need:
`:target = "isExternalLink ? '_blank' : '_self'"`

Comment (Ray2126) [reply components/content/ProseA.vue:5]: Nice catch! I've added this in

**Review by MeganSteenkamp — APPROVED** [review]

**Pull request merged**